### ISSUE TITLE
`tidy_players_stats()` when `players_id` have no NHL played game

### ISSUE THREAD

**Issue by stecaron** [jplecavalier/tidynhl#3]
The function `tidy_players_stats()` outputs an error when inputs a `player_id` for a player that has not played any NHL game. 

For a reproducible example, see the player Jared Knight (`player_id = 8475743L`).

<img width="389" alt="Screen Shot 2021-02-01 at 8 21 00 PM" src="https://user-images.githubusercontent.com/29443824/106538894-0500a880-64cb-11eb-8684-9ea37caaab78.png">

I would expect maybe an empty `data.table` as an output?

**Comment by jplecavalier** [comment]
Thanks for reporting, this should be resolved with 942c841. Reinstall the package from GitHub and try again, you should now have the following:

```
library(tidynhl)
tidy_players_stats(8475743L)
##  Empty data.table (0 rows and 50 cols): player_name,season_years,season_type,team_abbreviation,skater_games,skater_goals...
```

I will close the issue, please reopen if needed.

**Comment by stecaron** [comment]
Yep now it's working fine ! Thanks
